packages/custom_lint/lib/src/v2/server_to_client_channel.dart

```diff
@@ -409,6 +400,7 @@ $dependencies
 
   @override
   Future<void> init() async {
+    _packages = getPackageListForContextRoots(_contextRoots.roots);
```

Comment:
I moved this here as this can throw in the same way as the other initialization calls. The packages are only used during the creation of the temp project.